tailwind.config.js

```diff
@@ -22,6 +22,7 @@ export default {
           base: "#ffffff",
           inverse: "#000000",
           translucentGrey: "#00000080",
+          black: "#2B2313",
```

Comment:
this is not black right?

Reply:
Yeah actually let me rename it to dark instead. Done in 5d24736743594dba087578f039930e9ee7c206b0.

We don't yet have a fixed colour palette, but I think for the purpose of prototype we can defer that. This will change when we confirm the colour palette that we want to use.